bcipy/simulator/sim_parameters.json

```diff
@@ -0,0 +1,24 @@
+{
+  "sim_task": {
+    "value": "RSVP_COPY_PHRASE",
+    "section": "sim_config",
+    "readableName": "Simulation Task",
+    "helpTip": "Specifies Task to simulate",
+    "recommended_values": [
+      "RSVP_COPY_PHRASE",
+      "RSVP_CALIBRATION"
```

Comment:
I think we will only ever be simulating spelling tasks, so nothing that is a calibration. We could alternatively get this data from the TaskType (see the bcipy main module). Not all models will be relevant for all tasks, so that might be something to consider.